### PR TITLE
Validate chat message

### PR DESCRIPTION
**Pull request by AnandAili**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -142,7 +142,7 @@
         <dependency>
```
Comment (dimitriirtimid) [inline pom.xml]: Why are these changes in here? 
I suggest not doing 'manual' updates but start using the renovate PRs

File: src/main/java/io/localmotion/chatbox/ChatMessage.java [hunk 2]
```diff
@@ -28,7 +30,9 @@
 
     private final Date creationTime = new Date();
 
-    @NotBlank
+    @NotBlank(message = "Message must have at least " + SmokefreeConstants.ChatBox.MINIMUM_MESSAGE_LENGTH + " characters")
+    @Pattern(regexp = "^[A-Za-z0-9\\n!@&(),.?\": ]+$", message = "Please enter only allowed special charaxters: @&(),.?\": ")
```
Comment (dimitriirtimid) [inline src/main/java/io/localmotion/chatbox/ChatMessage.java]: Drop the regexp. We will trust (and test) the escaping of React.

File: src/test/java/io/localmotion/chatbox/ChatMessageTest.java [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+		message.setAuthor("dev-user-1");
+		message.setText("Hi Volunteers, Can this \n playground be smokefree");
+
+		Set<ConstraintViolation<ChatMessage>> violations = validatorFactory.getValidator().validate(message);
+		assertEquals(0, violations.size());
+
+	}
+
+	@Test
+	void should_notAllow_when_messageContainsNotAllowedCharacters() {
```
Comment (dimitriirtimid) [inline src/test/java/io/localmotion/chatbox/ChatMessageTest.java]: this one can go

File: src/test/java/io/localmotion/chatbox/ChatMessageTest.java [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+		message.setText(getMessage(4000));
+
+		Set<ConstraintViolation<ChatMessage>> violations = validatorFactory.getValidator().validate(message);
+		assertEquals("Message length must not exceed "+ SmokefreeConstants.ChatBox.MAXIMUM_MESSAGE_LENGTH +" characters", violations.iterator().next().getMessage());
+		assertEquals(1, violations.size());
+
+	}
+
+
+	private String getMessage(int maxLength) {
```
Comment (dimitriirtimid) [inline src/test/java/io/localmotion/chatbox/ChatMessageTest.java]: maxLength -> length

File: src/main/java/io/localmotion/chatbox/ChatMessage.java (hunk 2 above)
Comment (AnandAili) [reply src/main/java/io/localmotion/chatbox/ChatMessage.java]: I have removed regexp

File: src/test/java/io/localmotion/chatbox/ChatMessageTest.java (hunk 3 above)
Comment (AnandAili) [reply src/test/java/io/localmotion/chatbox/ChatMessageTest.java]: I agree I have disabled this test case.

File: src/test/java/io/localmotion/chatbox/ChatMessageTest.java (hunk 4 above)
Comment (AnandAili) [reply src/test/java/io/localmotion/chatbox/ChatMessageTest.java]: I agree , I renamed it

File: pom.xml (hunk 1 above)
Comment (AnandAili) [reply pom.xml]: there were so many renovate PR's. So I thought to have them in a single commit.
Agree It's good to handle them from PR's. Now onwards I will use renovate PR's

**Pull request merged**